test/OrchardCore.Tests/Recipes/RecipeFiles/recipe6.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "Recipe6",
+  "displayName": "Recipe 6",
+  "description": "This is a recipe for a bug to verify that the type-checking function works as expected",
+  "author": "Tony Han",
+  "website": "",
+  "version": "1.0.0",
+  "issetuprecipe": false,
+  "categories": [ "test" ],
+  "tags": [],
+  "variables": {
+    "messageContentItemId": "[js:uuid()]",
+    "now": "[js: new Date().toISOString()]"
+  },
+  "steps": [
+    {
+      "name": "feature",
+      "enable": [
+        "OrchardCore.Contents",
+        "OrchardCore.ContentTypes",
+        "OrchardCore.Title"
+      ]
+    },
+    {
+      "name": "ContentDefinition",
+      "ContentTypes": [
+        {
+          "Name": "Message",
+          "DisplayName": "Message",
+          "Settings": {
+            "ContentTypeSettings": {
+              "Creatable": true,
+              "Draftable": true,
+              "Versionable": true,
+              "Listable": true,
+              "Securable": true
+            }
+          },
+          "ContentTypePartDefinitionRecords": [
+            {
+              "PartName": "Message",
+              "Name": "Message",
+              "Settings": {
+                "ContentTypePartSettings": {
+                  "Position": "5"
+                }
+              }
+            },
```

Comment:
Are these really needed?

Reply:
I'll try it, I'm not sure, but I think this recipe is closer to the real scenario.

Reply:
Done.